blob/azureblob/azureblob.go

```diff
@@ -253,6 +270,16 @@ func DefaultSASToken() (SASToken, error) {
 	return SASToken(s), nil
 }
 
+// DefaultCloudEnvironment loads the desired Azure Cloud to target from
+// the AZURE_BLOB_DOMAIN environment variable.
+func DefaultCloudEnvironment() (CloudEnvironment, error) {
+	s := os.Getenv("AZURE_BLOB_DOMAIN")
+	if s == "" {
+		return CloudEnvironment("blob.core.windows.net"), nil // If unspecified, use default public domain.
```

Comment:
Nit: this would read simpler with `s = "blob.core.windows.net"` instead of a separate `return`.

Reply:
Or, could just leave it the empty string, since `openBucket` handles the default.